bitbots_bringup/launch/motion.launch

```diff
@@ -20,11 +20,8 @@
 
         </include>
         <node name="set_volume" pkg="bitbots_bringup" type="set_volume.sh" args="100%" />
-        <include file="$(find bitbots_buttons)/launch/buttons.launch"/>
```

Comment:
bitbots_butons and humanoid_league_speaker now won't appear in any launchfile, is that intended?

Reply:
They are now directly in the ros_control launch file with PR https://github.com/bit-bots/bitbots_lowlevel/pull/57